### PR TITLE
Applicant email verification template created

### PR DESCRIPTION
**Pull request by Shivam-Samant**
#1613

### CODE REVIEW

File: Modules/HR/Listeners/Recruitment/ApplicantEmailVerification.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    public function handle(ApplicationCreated $event)
+    {
+        $application = $event->application;
+        $applicant = $application->applicant;
+
+        $subject = Setting::where('module', 'hr')->where('setting_key', 'applicant_verification_subject')->first();
+        $body = Setting::where('module', 'hr')->where('setting_key', 'applicant_verification_body')->first();
+
+        $body->setting_value = str_replace(config('constants.hr.template-variables.applicant-name'), $application->applicant->name, $body->setting_value);
+        $verification_link = url('/') . '/applicantEmailVerification/' . encrypt($application->applicant->email) . '/' . $application->id;
```
Comment (GauravGusain98) [inline Modules/HR/Listeners/Recruitment/ApplicantEmailVerification.php:37]: I think we can use route here, instead of harcoding the URL.

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Minor feedback. Else looks good

File: Modules/HR/Listeners/Recruitment/ApplicantEmailVerification.php (hunk 1 above)
Comment (P4NK4J) [reply Modules/HR/Listeners/Recruitment/ApplicantEmailVerification.php]: Made changes in the latest commit, thanks

File: Modules/HR/Listeners/Recruitment/ApplicantEmailVerification.php [hunk 2]
```diff
@@ -34,7 +34,8 @@ public function handle(ApplicationCreated $event)
         $body = Setting::where('module', 'hr')->where('setting_key', 'applicant_verification_body')->first();
 
         $body->setting_value = str_replace(config('constants.hr.template-variables.applicant-name'), $application->applicant->name, $body->setting_value);
-        $verification_link = url('/') . '/applicantEmailVerification/' . encrypt($application->applicant->email) . '/' . $application->id;
+        $encrypedEmail = encrypt($application->applicant->email);
+        $verification_link = route('applicant.email.verification', [$encrypedEmail, $application->id]);
```
Comment (GauravGusain98) [inline Modules/HR/Listeners/Recruitment/ApplicantEmailVerification.php:38]: encrypted spelling is incorrect

**Review by GauravGusain98 — APPROVED** [review]
Looks good

**Pull request merged**